way_to_home/route/models.py

```diff
@@ -1 +1,73 @@
 """This module implements class that represents the route entity."""
+
+from django.db import models, IntegrityError
+from utils.abstract_models import AbstractModel
+from way.models import Way
+from place.models import Place
+
+
+class Route(AbstractModel):
+    """Model for Route entity."""
+
+    way = models.ForeignKey(Way, on_delete=models.CASCADE)
+    start_place = models.ForeignKey(Place, on_delete=models.CASCADE)
+    end_place = models.ForeignKey(Place, on_delete=models.CASCADE)
+    time = models.TimeField()
+    transport_id = models.PositiveIntegerField()
```

Comment:
```suggestion
    transport_id = models.PositiveIntegerField()
```
It can be empty, so set `null=True`

Reply:
Done